eclass-writing/text.xml

```diff
@@ -114,8 +113,10 @@ batch push them at once where possible.
 <title>Removing Eclasses</title>
 <body>
 
-<p>No longer used eclasses may be removed from the tree, but developers must
-adhere to the following process:</p>
+<p>
+No longer used eclasses may be removed from the tree, but developers must
+adhere to the following process:
+</p>
```

Comment:
I wonder if we should require a `@DEPRECATED` tag exist for some period beforehand (at the very least a month before removal, like we do for `@DEAD`), given otherwise nobody gets a pkgcheck warning about it...

Reply:
It's mentioned in the `<note>` below the list.

Maybe we could clarify "Do this at least 30 days before removing the eclass." instead of "Do this for a period before removing the eclass."?

Reply:
That would WFM.